app/models/edition.rb

```diff
@@ -88,6 +89,7 @@ def change_note_html
   def draft_copy
     dup.tap do |e|
       e.change_note = nil
+      e.change_reason_for_change = nil
```

Comment:
I think this method along with #ensure_draft_exists can go can't it? They're no longer used?